_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "New OmbuLabs Products"
+date: 2016-02-04 18:35:00
+categories: ["Design"]
+author: "cecilia"
+---
+
+As part of Christmas presents for the Ombu Labs' team, we worked on a series of products. We want to share with you the process and its final result.
+
+<!--more-->
+
+### It all started with an idea
+
+We decided to make 3 products: a t-shirt, a notebook and a tote bag. As it happens with every product, it starts with an idea, then some sketches, its design and, finally, its creation.
+
+### Products with design
+
+For the notebook, we chose a quote by Otl Aicher’s: “Good Art Inspires, Good Design Motivates”. Here you can see part of the design process, with some tweaks and final adjustments.
+
+![Notebook Design Process](http://www.ombushop.com/blog/wp-content/uploads/2016/02/cuaderno-proceso.png)
```

Comment:
@cecilia-ombulabs Please don't "hotlink" to ombushop.com, place the images in this project. Same goes for the rest of the images.